pkg/dashboard/server.go

```diff
@@ -52,7 +52,14 @@ func (s *Server) StartServer(ctx context.Context, cancel context.CancelFunc) (st
 	api := NewRouter(cancel, data, s.Debug)
 	done := s.startBackgroundServer(api, ctx)
 
-	return "http://" + s.Address, done, nil
+	basePath := strings.TrimSpace(os.Getenv("HD_BASE_PATH"))
```

Comment:
Instead of calculating base path here, make it a field in `Server` struct and pass the value from [main.go](main.go). Because this option looks exactly like server start config option. This way we have single place that calculates the option value (main.go) and then others use it from field/variable.

Value of this option would then also go into `NewRouter` call here.